src/nodes.js

```diff
@@ -228,10 +228,14 @@ module.exports = {
     const args = join(concat([",", line]), path.call(print, "body", 1));
 
     // Hate, hate, hate this but can't figure out how to fix it.
-    return group(ifBreak(
-      concat([command, " ", align(command.length + 1, args)]),
-      concat([command, " ", args])
-    ));
+    return group(
+      ifBreak(
+        isIdentAccessModifier(path.getNode().body[0])
```

Comment:
cc @kddeisz This PR isn't quite ready yet; But would like to get your thoughts on the above if possible 🤞 

I was planning on handling the formatting of access modifiers separately:

Input:

```ruby
private def private_foo
          true
        end
```

New Output:

```ruby
private def private_foo
  true
end
```

The above approach seems to be roughly what I want, but I'm on the fence about the formatting of:

```ruby
custom_method_helper def foo
                       true
                     end
```

From the definition of https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/cop/layout/def_end_alignment.rb it should be formatted as:

```ruby
custom_method_helper def foo
  true
end
```

Just wanting to verify that this is the output we'd also want for plugin-ruby?

And if so, I wonder if you have any thoughts on what the best way to approach this is? I'm envisioning the implementation would also have to handle something like:

```ruby
private custom_method_helper def foo
  true
end
```

Any insights would be appreciated! 🎉

Reply:
I ran the RipperJS AST for the final example through graphviz and went with a recursive approach for working out if the command has a def statement immediately, or if has a nested def command:

![image](https://user-images.githubusercontent.com/1271782/53379677-742a0580-3962-11e9-88a8-c3456a8fa7a4.png)

Now the output is as expected:

```
private custom_method_helper def foo
  true
end
```